### DISCUSSION TITLE
Some icons are not displayed

### DISCUSSION THREAD

**Discussion by pwsandoval** [microsoft/terminal#14867] [Q&A] [answered]
Windows terminal version: 1.16.10262.0
OS Version: Windows 11 Pro (OS Build 22000.1574)

I have seen today that in the taskbar when hovering or right clicking the ubuntu icon is not shown (the powershell one can be seen):

![icon task](https://user-images.githubusercontent.com/15174704/219765639-b52fabf2-3595-48d5-861e-fda07d884752.png)

Within the program if it is displayed correctly:

![icon-wt](https://user-images.githubusercontent.com/15174704/219766186-4488ffbb-7f97-46ad-bf86-be6e3c0ba9e2.png)

My settings are this:

```json
"profiles": {
    "defaults": { ... },
    "list": [
    {
        "commandline": "pwsh.exe -nologo",
        "guid": "{574e775e-4f2a-5b96-ac1e-a2962a402336}",
        "hidden": false,
        "name": "pwsh",
        "source": "Windows.Terminal.PowershellCore"
      },
      {
        "guid": "{59b72ad2-09f7-55ec-af87-65d062af8472}",
        "hidden": false,
        "tabTitle": "bash",
        "suppressApplicationTitle": true,
        "name": "bash",
        "source": "CanonicalGroupLimited.Ubuntu20.04LTS_79rhkp1fndgsc"
      }
    ]
```

Thank for your time.

**Comment by zadjii-msft** [comment, marked as answer]
This is tracked in #10552. The Canonical Ubuntu distro profiles all have URLs that point at web resources, and the Jumplist can't load those.

**Comment by yoel-du** [comment]
I have been troubled by this issue for the past 6 months as well.

![image](https://github.com/microsoft/terminal/assets/38520794/ea5d3c6c-e96e-486e-a708-352fa1c38a8e)
![image](https://github.com/microsoft/terminal/assets/38520794/34ed2255-485d-40f6-be5e-df1f9fdd0984)
